numpy/_core/_add_newdocs.py

```diff
@@ -4713,6 +4713,86 @@
     its memory, in which case you can traverse ``a.base`` for a memory handler.
     """)
 
+add_newdoc('numpy._core._multiarray_umath', '_array_converter',
+    """
+    _array_converter(*array_likes)
+
+    Helper to convert one or more objects to arrays.  Integrates machinery
+    to deal with the ``result_type`` and ``__array_wrap__``.
+
+    The reason for this is that e.g. ``result_type`` needs to covert to arrays
+    to find the ``dtype``.  But converting to an array before calling
+    ``result_type`` would incorrectly "forget" whether it was a Python int,
+    float, or complex.
+    """)
+
+add_newdoc('numpy._core._multiarray_umath', '_array_converter', ('scalar_input',
+    """
+    A tuple which is ``True`` if the original inputs were scalars and
+    ``False`` otherwise.  Scalars are all objects which are coerced to a
+    0-D array but are not already arrays or converted via one of the protocols.
+
+    .. note::
```

Comment:
My sense would be to delete this note: it probably adds more confusion than it solves...